recipe/meta.yaml

```diff
@@ -6,8 +6,8 @@ package:
 
 source:
   fn: notebook-{{ version }}.tar.gz
-  url: https://github.com/jupyter/notebook/archive/{{ version }}.tar.gz
-  sha256: edb7ef8153e5ca781f21c7feb50b3316d515b0d8861ba0e45a00280166214a7f
+  url: https://pypi.python.org/packages/source/n/notebook/notebook-{{ version }}.tar.gz
+  md5: 3a72bb93f6f01caec69e5e65718f5f7f
```

Comment:
👎  for using PyPI. 😕 

github is the canonical source IMO. That said, if there is good reason to use PyPI, I'm all for it.